account_invoice_pricelist_sale/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Account Invoice Pricelist - Sale",
+    "summary": "Module to fill pricelist from sales order in invoice.",
+    "author": "Therp BV,GRAP,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/account-invoicing",
+    "category": "Sales",
+    "version": "15.0.1.0.1",
```

Comment:
```suggestion
    "version": "15.0.1.0.0",
```
Can you please commit this change?